frontend/src/scenes/onboarding/Onboarding.tsx

```diff
@@ -316,6 +317,23 @@ const SessionReplayOnboarding = (): JSX.Element => {
             value: true,
             visible: false,
         },
+        {
+            type: 'select',
```

Comment:
does this add a data-attr or something unique that we can use to track people's choices here?